02. 1, 2차원 배열 탐색/07. 봉우리/김민영/index.js

```diff
@@ -1,6 +1,19 @@
 function solution(arr) {
   let answer = 0;
 
+  for (i = 0; i < arr.length; i++) {
+    for (j = 0; j < arr[0].length; j++) {
+      let top = i > 0 ? arr[i - 1][j] : 0;
+      let bottom = i < arr.length - 1 ? arr[i + 1][j] : 0;
+      let left = j > 0 ? arr[i][j - 1] : 0;
+      let right = j < arr[0].length - 1 ? arr[i][j + 1] : 0;
```

Comment:
`top`이랑 `bottom`은 `undefined`에서 프로퍼티를 참조할 위험이 있어서 조건 잘 거신 거 같은데 `left`랑 `right`에서 `arr[i]`는 있는 요소니까 short-circuit으로 `falsy` 처리만 하면 어떨까 싶어요!
```suggestion
      let left = arr[i][j - 1] || 0;
      let right = arr[i][j + 1] || 0;
```

Reply:
`j`가 `0`이거나 `arr[0].length - 1`일 때를 대비해서 조건을 걸었는데 없어도 되는 건가요 ..?!